lib/Image/Adapter/Imagick.php

```diff
@@ -325,7 +325,6 @@ private function hasAlphaChannel(): bool
             for ($i = 0; $i < $width; $i++) {
                 for ($j = 0; $j < $height; $j++) {
                     $pixel = $this->resource->getImagePixelColor($i, $j);
-                    // @phpstan-ignore-next-line - phpstan expects bool, but actually doc says int
```

Comment:
tests failin because of  the [unreleased ](https://github.com/phpstan/phpstan-src/pull/2020)changes, so waiting for 1.9.3

Reply:
@kingjia90 We can set the phpstan version to 1.9.2 in the composer.json till the version 1.9.3 is released.

Reply:
you mean 1.9.x-dev?

Reply:
@kingjia90 I meant set to version 1.9.2 and leave the comment. But without comment with 1.9.x-dev could also work :)